### PR TITLE
Add: Send Celery log data to the Docker logs.

### PR DESCRIPTION
**Pull request by bjones1**
Docs: Prettify a bit.

### CODE REVIEW

File: docker/docker_tools_misc.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def _start_servers(dev: bool) -> None:
     xqt(
         'sudo -u www-data env "PATH=$PATH" "REDIS_URI=$REDIS_URI" '
         "poetry run celery --app=scheduled_builder worker --pool=threads "
-        "--concurrency=3 --loglevel=info &",
+        "--concurrency=3 --loglevel=info --logfile=/dev/stdout &",
```
Comment (bnmnetp) [inline docker/docker_tools_misc.py:91]: celery probably will not get restarted very often but the redirect I used below `/proc/1/fd/1` ensures that the log output continues to go to the docker log even after the server is restarted.  Whereas a simple redirect to stdout does not.

Comment (bjones1) [reply docker/docker_tools_misc.py:91]: Thanks! I'll update that. Do you have any docs / links to discuss why this works? I'd like to include that if so. It's a neat technique.

Comment (bnmnetp) [reply docker/docker_tools_misc.py:91]: I found it while searching for a solution.  It is discussed in several places.  Such as https://github.com/moby/moby/issues/19616 and others.

Comment (bjones1) [reply docker/docker_tools_misc.py]: Great, thanks! I'll add that link.

**Comment by bjones1** [comment]
Ignore the test failure -- that's Poetry complaining that dependencies are outdated. The latest check ran on a docs-only commit; the previous check (on code changes) passed.

**Comment by bjones1** [comment]
Whoops, I realized I committed changes to `pyproject.toml` in this PR -- I meant that to be in a separate PR...

**Comment by bnmnetp** [comment]
I think thats fine, we wanted that in there anywhere.

**Pull request merged**